src/main/java/edu/cornell/kfs/module/ar/batch/service/impl/CuCustomerLoadServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package edu.cornell.kfs.module.ar.batch.service.impl;
+
+import edu.cornell.kfs.module.ar.document.validation.CuCustomerRule;
+import org.kuali.kfs.kns.document.MaintenanceDocument;
+import org.kuali.kfs.module.ar.batch.report.CustomerLoadBatchErrors;
+import org.kuali.kfs.module.ar.batch.service.impl.CustomerLoadServiceImpl;
+
+public class CuCustomerLoadServiceImpl extends CustomerLoadServiceImpl {
+    
+    @Override
+    protected boolean validateSingle(MaintenanceDocument maintDoc, CustomerLoadBatchErrors batchErrors, String customerName) {
+        boolean result = true;
+
+        //  get an instance of the business rule
+        CuCustomerRule rule = new CuCustomerRule();
```

Comment:
Why do we need CuCustomerLoadServiceImpl?  Looks like the only difference is we are using CuCustomerRule instead of CustomerRule.  Since CuCustomerRule extends CustomerRule, it should caste correctly in the parent, right?

Reply:
Since the BR class is not retrieved from DD I am not sure how it would know to cast to CuCustomerRule. Perhaps I am missing something?